### DISCUSSION TITLE
Fallout - Pip Boy (portrait) [3.5"]

### DISCUSSION THREAD

**Discussion by geekm0nkey** [mathoudebine/turing-smart-screen-python#291] [Themes]
Here is another theme I came up with. As always, PSD is included should you want to change things around. The preview looks dark because I added a photo filter overall to darken it as my screen has a blueish tint and this is to counter that. You can disable that layer in PS to brighten it up. 

**Updated** - Made some minor adjustments to the layout.
**Updated 2** - More layout adjustments, additions and tweaks.

![screencap](https://github.com/mathoudebine/turing-smart-screen-python/assets/35175147/cd36e989-30dc-4374-872c-7c82b0f3362c)

**Download**
[Fallout.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/12073956/Fallout.zip)

**Comment by ctheof** [comment]
![screencap](https://github.com/mathoudebine/turing-smart-screen-python/assets/4469212/d00409dd-0cb0-4e32-a2f4-2e29ec3dd54c)
Love this theme! My desk has a Fallout vibe to it at the moment and this fits perfectly
I made a tweak and added time/date to the theme. Does anyone know what the [] character is next to the minutes?

Also is there a way to display the RAM free and RAM used in GB instead of MB?

**Comment by mathoudebine** [comment]
Amazing theme! Thanks @geekm0nkey!

**Comment by KarmusKirin** [comment]
A wonderful theme, but I couldn't find the PDF in the files. I wanted to try animating the puppet just like in this theme: https://www.youtube.com/watch?app=desktop&v=9NmUXEhfLTM. I believe that if we use it as input parameters, it should work, although I haven't found the files for that yet. Nonetheless, it's worth researching.

**Comment by itsprstar** [comment]
Nice!, one question tough, For what base (.data file) theme is this skin made?

**Comment by StichSince1996** [comment]
Olá! Como faço para transformar isso em um tema no formato .data?

**Comment by Yehuda-Bergstein** [comment]
Amazing theme!!
Is there a way to add Ghz reading of the cpu?

**Comment by SuperWaper** [comment]
Hello, no % and temperature was displayed, on turing smart screen 3.5 any idea ?

**Comment by Sluggix** [comment]
Hello everyone!
After trying to add the theme using Python (without success — I’m really not a pro in that area), I spent quite a bit of time figuring out a workaround using the USB Monitor software… and it finally worked! 🎉
So no, it doesn’t display exactly the same info — I had to adapt.
I used the rogwhitetexte theme as a base, then imported my background via the Theme Editor, and finally adjusted the text positions and colors manually.

It was a long and tedious process 😅
One thing to note: if you hide a custom field like "CPU", it won’t reappear — you’ll have to start over from scratch. Be careful!

Unfortunately, I can’t share the .data file here, and I’m not sure if it’s even possible. Let me know if you have a solution for that!

![Capture d'écran 2025-05-21 174638](https://github.com/user-attachments/assets/71e33b8b-5f09-4030-9396-5f8d094fdcb2)
![Capture d'écran 2025-05-21 174659](https://github.com/user-attachments/assets/cfa94510-5fc7-4803-9d2f-e25512882f0e)
![Capture d'écran 2025-05-21 175356](https://github.com/user-attachments/assets/180869f2-12c7-4476-bfcb-da0939b221
![WhatsApp Image 2025-05-21 à 17 58 32_11aa786a](https://github.com/user-attachments/assets/af7d13cd-bea8-4287-b0c4-6d9c3f66a93d)
ed)

**Comment by shabudabu** [comment]
fixed theme in atach
[Fallout.zip](https://github.com/user-attachments/files/20429409/Fallout.zip)

**Reply by Sluggix** [reply]
Thank you very much. I just tried using the Windows software turing-system-monitor-3.9.3.exe. I added the theme to the files, but some data is missing — GPU and CPU temperatures, and GPU usage percentage. However, I ran the test using python/external/LibreHardwareMonitor/test_librehardwaremonitor.py, and the values are showing up correctly. 
![Capture d'écran 2025-05-25 135455](https://github.com/user-attachments/assets/e24c19e1-a73e-4e68-a17f-3830a5ab20e4)
![Capture d'écran 2025-05-25 134336](https://github.com/user-attachments/assets/2a6b006b-fb48-4e57-aeb9-a3c9131e72c3)

**Reply by agentgerbil** [reply]
I'm also having the same problem.

**Reply by MichiderKluge** [reply]
Same here

**Reply by jbovatsek** [reply]
I think OP accidentally attached the same broken attachment.

The GPU interval is in the wrong place and the CPU temp interval is missing.  Here's a working one, or you can modify your own by comparing to a working theme.

[Fallout.zip](https://github.com/user-attachments/files/21540101/Fallout.zip)

**Reply by MichiderKluge** [reply]
Worked perfect, thank you

**Reply by mathoudebine** [reply]
For anyone wondering how to fix it: https://github.com/mathoudebine/turing-smart-screen-python/wiki/Troubleshooting#all-os-my-cpugpuother-hardware-is-not-displayed--some-data-are-missing

**Comment by geekm0nkey** [comment]
I recently had to replace my computer. my new one is a Ryzen 9 AI with integrated GPU and now it does the same for me. I had to pick a new skin. Seems some new processors are not supported yet.

to be clear the new skin didn't have some sensor data displayed not that it worked with other skins.

I ended up using this one.

![image](https://github.com/user-attachments/assets/0558b852-0627-4ad8-a375-b8dca79b856a)
